src/time_series_formats.jl

```diff
@@ -25,6 +25,33 @@ function read_time_series(file_path::AbstractString, component_name = nothing; k
     return read_time_series(get_time_series_format(file), file, component_name; kwargs...)
 end
 
+"""
+Return a TimeDataContaier from a CSV file.
+
+Pass component_name when the file does not have the component name in a column header.
```

Comment:
How do you envision attaching this to a specific component?

Reply:
Can we use a similar pointers file but instead of going from columns -> component it will be file -> component?